### ISSUE TITLE
[Social Media] DevNexus MicroProfile Content

### ISSUE THREAD

**Issue by m-reza-rahman** [microprofile/microprofile-marketing#758]
Might be good to promote the MicroProfile sessions at DevNexus: https://devnexus.com/presentations.

**Comment by m-reza-rahman** [comment]
Tagging @rstjames for awareness.

**Comment by aeiras** [comment]
Hey @m-reza-rahman, 

This ticket for you as the Steward of Feb is great for awareness of what is coming next. However, MP media doesn't create social media with respect to sessions without the speaker submitting his/her ticket and providing the information required directly in the ticket as stated on the https://microprofile.io/resources/ under[ Submit a Marketing Issue. ](https://github.com/eclipse/microprofile-marketing/wiki/MP-DevRel---Branding-Resources---Community-Support)

The Steward's job is to pull in those speakers on any manner they might choose and remind them of the available social highlight. Yet it is a volunteer activity not actually required by the speaker, so without the speaker's ticket, there is no media.

Since I am January's steward, you can see how I chose to pull the speakers' directly via the MP Community Out reach section in the minutes https://docs.google.com/document/d/1tDum7Bm6LtFXQbcUfxw6DTuVkhjuaXmqu1fpaKaTIto/edit#

It is up to you to follow up either via private or whatever you see fit in Feb. Ideally, there are at least 2 media messages going out for each session and a general for DevNexus MicroProfile sessions (The general media message can be created by you, the Steward of Feb, in it, we tend to highlight all the MP sessions within 1 frame) .

I hope this is helpful!

**Comment by ederks85** [comment]
Hi @aeiras, I see your proposal to submit my/Josh's session for DevNexus 2023 via Marketing. Naturally I need to provide information but the Marketing Issue template seems to be empty aside from a template title "[Social Media] Name of Event - Talk]". Am I overseeing something regarding a template for this? Could you otherwise please specify what kind of information is required in the marketing issue?

**Comment by m-reza-rahman** [comment]
Hi @ederks85, I think it's fairly simple. All you need to do is provide enough information for @rstjames to formulate a tweet. We should do that fairly soon to help DevNexus promotion. /cc @Emily-Jiang

**Comment by m-reza-rahman** [comment]
> Ideally, there are at least 2 media messages going out for each session and a general for DevNexus MicroProfile sessions (The general media message can be created by you, the Steward of Feb, in it, we tend to highlight all the MP sessions within 1 frame) .

OK. I suspect both @Emily-Jiang and @ederks85/@juneau001 will follow up on their own (if not I will reach out). In the meantime, can we get out the following more general tweet?

> Want to find out the latest on MicroProfile and connect with some key community leaders? DevNexus in April is a great place to do that and more! Check out the sessions and register today. https://devnexus.com/presentations

I am sure @rstjames could make up a suitable social card, but something like this is also fine: https://twitter.com/devnexus/status/1617931582861606913/photo/1.

**Comment by aeiras** [comment]
> Hi @aeiras, I see your proposal to submit my/Josh's session for DevNexus 2023 via Marketing. Naturally I need to provide information but the Marketing Issue template seems to be empty aside from a template title "[Social Media] Name of Event - Talk]". Am I overseeing something regarding a template for this? Could you otherwise please specify what kind of information is required in the marketing issue?

@ederks85, 

Thank you for pointing out the issue with the documentation, something was lost early this month when the wiki was imported to the resources page in MP.io. I have taken the opportunity to adjust a few things within the wiki as times. 
Try again https://github.com/eclipse/microprofile-marketing/wiki/MP-DevRel---Branding-Resources---Community-Support

@m-reza-rahman is correct in the follow up. Each session will need its own git issue. Ideally, you and Josh could write 2 different descriptions for the session given that your session could be scheduled 4 times if you choose to submit 2 months in advance and not just 2 weeks before the session. 

Cheers,

**Comment by ederks85** [comment]
Thanks for the support, the template really helps!! I have submitted a ticket :)

**Comment by aeiras** [comment]
@m-reza-rahman, 

The two sessions at Devnexus are now covered:
https://github.com/eclipse/microprofile-marketing/issues/762
https://github.com/eclipse/microprofile-marketing/issues/761

As the MP marketing Steward for Feb, you could create a general message highlighting the 3 MicroProfilers and DevNexus if it is valuable. Using this ticket is okay, or closing it and creating a specific ticket focusing on the general message might be cleaner.  There are no wrongs in the process. 

--- 

As the Feb Steward, a general Steward collaborative git ticket where others in the community can communicate with you and you with us is recommended.

**Comment by m-reza-rahman** [comment]
How about if we just moved this forward: https://github.com/eclipse/microprofile-marketing/issues/758#issuecomment-1402296785? Or did you have something else in mind?

**Comment by aeiras** [comment]
> How about if we just moved this forward: [#758 (comment)](https://github.com/eclipse/microprofile-marketing/issues/758#issuecomment-1402296785)? Or did you have something else in mind?

Hey Reza, 
For a potential general message, for example: join [named MicroProfilers Twitter handles + DevNexus ] using https://reg.connectevents.io/ConnectEvents/devnexus2023/ dates might be better suited to the example you provide with 50 % discount.  DevNexus' focus is to enable access to all people with an incredible discount. AJUG is truly **amazing**. MP focuses on highlighting the 3 MicroProfilers and DevNexus in the general message. 

Ryan will need the exact wording and direction of what to put in the media card. 
The more direct and specific we are on git marketing tasks,  especially when highlighting MicroProfilers at events, the better. 

In short, if it takes too much work for others to figure out, it is not worth working on it b/c it cannot be scaled or replicated. 

I hope this follow-up is helpful. We don't have it (this feedback on general social sessions media) written under Steward's documentation. If you think it is available you could choose to update that wiki. All up to you as you navigate coverage and support in MP.

**Comment by m-reza-rahman** [comment]
OK. How is the following (it is at the max character count that Twitter allows)? Is it sufficient to move it forward?

> Please join @emilyfhjiang, @edwinderks and @javajuneau at DevNexus April 4–6 to catch up with the latest on MicroProfile. DevNexus is founded by MicroProfile member Atlanta JUG and is offering a thoughtful 50% discount to folks affected by recent layoffs. https://reg.connectevents.io/ConnectEvents/devnexus2023/

With regards to the card, I honestly think this is just fine: https://twitter.com/devnexus/status/1617931582861606913/photo/1. If time permits, something not too crowded incorporating the DevNexus logo, the MicroProfile logo, speaker names/photos/session titles is ideal. I think think is a good example: https://twitter.com/JakartaEE/status/1618217121837613057/photo/1.

**Comment by aeiras** [comment]
@m-reza-rahman, good draft. 

I added @ on mentions b/c media volunteers only get notified by mentions with @. 

**Join @Java_Champions @emilyfhjiang, @edwinderks, and @javajuneau at @devnexus & catch up with the latest on #MicroProfile. 
#DevNexus is founded by MicroProfile member @atlantajug, who offers a thoughtful 50% discount to folks affected by recent layoffs.** 

**https://reg.connectevents.io/ConnectEvents/devnexus2023/**

---

@rstjames, 

Thank you for collaborating with Reza (next week to close this ticket) to get out 2 Feb & 2 March media messages highlighting MP speakers, MP, and Devnexus/AtlantaJUG.

**Comment by m-reza-rahman** [comment]
Great! LMK if anything else is needed?

**Comment by rstjames** [comment]
@aeiras I just added the other two sessions to hootsuite and they go out 2 times... it would be repeating almost teh same info again if we do this one. Thoughts?

**Comment by aeiras** [comment]
Hey @rstjames, 

The general message is different that the 2 sessions' posts. 
Please pull Reza for clarification. Thanks for unblocking the task with Q&A.

**Comment by aeiras** [comment]
Oh and i re-open the session ticket b/c I noticed 2 schedules instead of 4. 
The idea given the 2-month warning is to push the sessions 4 times in the next 2 months instead of the traditional 2 times if git arrives within 2 weeks of the event. 

Please lets check that Emily and Edwin's are adjusted to match the 4 posts. Thank you, Ryan!

**Comment by rstjames** [comment]
My mistake I read the date wrong and thought it was this month... so I will make this ticket and make sure they go out 3-4 time before the sessions

**Comment by rstjames** [comment]
@m-reza-rahman Trying to think of the SM image for this post. It has two message. Check out the MP people at the conference and that there is a discount for people who lost there jobs... What is the main push? 
We already posting about the MP session so should we push more the Discount/ help?

**Comment by m-reza-rahman** [comment]
Hi Ryan, is there anything you can think of to make the wording better? My thought is to run this tweet first as more of an overview and then subsequently the session specific tweets. Hopefully it is highlighting different enough things. I’m personally not a big fan of repeating tweets too many times but I am OK with whatever you think works best.

With regards to the card, I think we can focus on putting it all together from a MicroProfile perspective rather than the discount per se. For example, I think it’ll be cool to list both sessions, all speakers, the AJUG logo, the MicroProfile logo and the DevNexus logo. Hopefully it conveys the message that the technology has a good position at DevNexus.

**Comment by rstjames** [comment]
All set. Put out it out on SM 2 times before the conference
 
![MicroProfile-devnexus-2023](https://user-images.githubusercontent.com/12132146/217223287-5e822946-080a-4a71-bff9-f2979da92011.jpg)
